contract/deploy/dnsSample.ts

```diff
@@ -9,6 +9,14 @@ const PUBLIC_RESOLVER = process.env.PUBLIC_RESOLVER as string
 const REGISTRAR_CONTROLLER = process.env.REGISTRAR_CONTROLLER as string
 const PRICE_ORACLE = process.env.PRICE_ORACLE as string
 
+const sleep = async (milliseconds) => {
```

Comment:
type

Reply:
done in https://github.com/polymorpher/ens-deployer/pull/15